app/model_manager.py

```diff
@@ -143,7 +146,7 @@ def recursive_search_models_(self, directory: str, pathIndex: int) -> tuple[list
 
         return [{"name": f, "pathIndex": pathIndex} for f in result], dirs, time.perf_counter()
 
-    def get_model_previews(self, filepath: str) -> list[str]:
+    def get_model_previews(self, filepath: str) -> list[str | BytesIO]:
```

Comment:
Are you sure `BytesIO` return type will pass `os.path.isFile` check?

https://github.com/comfyanonymous/ComfyUI/blob/c441048a4f885ba8f3bc0ce959b5df034d63368a/app/model_manager.py#L62-L65

Reply:
I'm a bit confused. The code in my PR modifies this check, and will only call `os.path.isfile` if the value given is a `str` instance.